### ISSUE TITLE
Rollback?

### ISSUE THREAD

**Issue by scitbiz** [mathielo/dst-dedicated-server#8]
Can we rollback?

**Comment by mathielo** [comment]
Yes, as a server admin you can rollback when you are connected to the server. Press `<Tab>` to display the player list, then click on `...` (Server Actions):
![step1](https://user-images.githubusercontent.com/4922271/41819741-0143410c-77c6-11e8-8fe5-3d1ccd9b296e.jpg)

Then select _Rollback_:
![step2](https://user-images.githubusercontent.com/4922271/41819747-20facdda-77c6-11e8-91e5-a8286b7cc6f0.jpg)
![step3](https://user-images.githubusercontent.com/4922271/41819748-2111ab5e-77c6-11e8-816c-04875198f91e.jpg)

The Steam account that generates the Cluster Token is automatically granted admin rights on a server. You can also add more admins by creating this file: `DSTClusterConfig/adminlist.txt` and adding Steam IDs to it (one per line).

**Comment by scitbiz** [comment]
Thank you. I didn't see that menu. My bad
